Detectors/Base/src/GRPGeomHelper.cxx

```diff
@@ -131,6 +131,10 @@ bool GRPGeomHelper::finaliseCCDB(ConcreteDataMatcher& matcher, void* obj)
   if (mRequest->askMatLUT && matcher == ConcreteDataMatcher("GLO", "MATLUT", 0)) {
     LOG(info) << "material LUT updated";
     mMatLUT = o2::base::MatLayerCylSet::rectifyPtrFromFile((o2::base::MatLayerCylSet*)obj);
+    if (getenv("ALICEO2_MATBUT_VOXEL")) {
+      // the object is already const here ... so need to const_cast to finish initialization
+      const_cast<o2::base::MatLayerCylSet*>(mMatLUT)->initLayerVoxelLU();
+    }
```

Comment:
Cannot this go directly to the rectifyPtrFromFile method, so that the optimization is available also to the analysis which does not use GRPGeomHelper

Reply:
sure